rllib/connectors/action/pipeline.py

```diff
@@ -19,10 +21,17 @@
 class ActionConnectorPipeline(ConnectorPipeline, ActionConnector):
     def __init__(self, ctx: ConnectorContext, connectors: List[Connector]):
         super().__init__(ctx, connectors)
+        self.timers = defaultdict(_Timer)
+
+    def reset(self, env_id: str):
+        self.timers.clear()
 
     def __call__(self, ac_data: ActionConnectorDataType) -> ActionConnectorDataType:
         for c in self.connectors:
-            ac_data = c(ac_data)
+            timer = self.timers[str(c)]
+            with timer:
+                ac_data = c(ac_data)
+            timer.push_units_processed(1)
```

Comment:
Went back to look at the implementation for this and found that it is only needed for throughput measurements. Since the mean time is not calculated over the units processed but over the number of timings. Thanks!